lib/plugins/aws/package/compile/events/websockets/lib/stage.js

```diff
@@ -31,7 +31,7 @@ module.exports = {
           // DeploymentId is generated at deployment.js file
           StageName: this.provider.getStage(),
           Description:
-            this.serverless.service.provider.websocketsDescription || 'Serverless Websockets',
```

Comment:
We should still support the old syntax, but checking the new one first when resolving the property

Reply:
Ok, I will add the old back to those four areas. Thanks.